### PR TITLE
Deleted submittal button from Report and ACC pages.

### PR DESCRIPTION
**Pull request by tanvirp10**
Deleted splitter from Report and ACC pages.
Cleaned up unused code.

### CODE REVIEW

**Review by dougwaldron — CHANGES_REQUESTED** [review]
The form changes all look good, but there's an issue with the SQL that will causes error when you run it. (Go ahead and run it from Visual Studio and try saving a new report or editing an existing one so you can see what the errors look like.)

The issue is that the parameters in the SQL statements and parameter arrays have to match up. Everywhere that you deleted a `SqlParameter` (the variables that start with "@") but didn't update the corresponding SQL statement will cause the database to throw an error. Here's an example -- the SQL statement still expects `@sub` to exist in the parameter list, but it's not there anymore:
image

When working on the fix for this, just remember that the `strSubmittalNumber` column still exists in the database. But going forward, we only want it to be set to "1" instead of letting the user increase it.

File: IAIP/SSCP/SSCPEvents.vb [hunk 1]
```diff
@@ -1969,57 +1814,6 @@ Public Class SSCPEvents
...
-                rdbACCAllDeviationsReportedYes.Enabled = True
-                rdbACCAllDeviationsReportedNo.Enabled = True
-                rdbACCAllDeviationsReportedUnknown.Enabled = True
-                rdbACCResubmittalRequestedNo.Enabled = True
-                rdbACCResubmittalRequestedUnknown.Enabled = True
-                rdbACCResubmittalRequestedYes.Enabled = True
-                DTPACCPostmarked.Enabled = True
-                chbACCReceivedByAPB.Enabled = True
-                dtpAccReportingYear.Enabled = True
-            End If
```
Comment (dougwaldron) [inline IAIP/SSCP/SSCPEvents.vb:1817]: Another thing I meant to mention. In this section, the UI gets updated. Previously, the update depended on the submittal value. Now the submittal value will always be "1", but the UI still needs to be updated.

Comment (dougwaldron) [reply IAIP/SSCP/SSCPEvents.vb:1817]: Sorry, that code section wasn't quite right. It starts here for the ACC, and also there's a similar one for the Reports:
image

**Comment by tanvirp10** [comment]
Sorry about that, I guess it just slipped my mind!

**Review by dougwaldron — APPROVED** [review]

**Pull request merged**